src/tinypeel/Peeling/PeelingUpdates.py

```diff
@@ -241,64 +235,64 @@ def updateSeqError_ind(counts, errors, refReads, altReads, genoProbs):
 #
 
 
-def updateSeg(peelingInfo):
```

Comment:
@XingerTang these edits to `getNewton()` and related functions in this `PeelingUpdates.py` file should not be in this PR - best to keep each PR simple and doing mostly one thing - polishing flake8 and tests is fine, but you are doing something else here,

Reply:
@gregorgorjanc I comment this part because this whole part of the code is broken (more details about it in the report I wrote), and `updateSeg()` is trying to call a function `setupTransmission()` which is not defined so the flake8 complained about this. So, it is flake8 related but I did do some extra here, I can uncomment the functions other than `setupTransmission()` if it is better.

Reply:
> @gregorgorjanc I comment this part because this whole part of the code is broken (more details about it in the report I wrote), and `updateSeg()` is trying to call a function `setupTransmission()` which is not defined so the flake8 complained about this. So, it is flake8 related but I did do some extra here, I can uncomment the functions other than `setupTransmission()` if it is better.

I see! Hmm, odd, BUT good find!

Please open an issue so we have a history of such changes in the repository with a clear statement of what we should do and then PRs solving the issue are linked to the issues through "hash-links".

Then let's solve this as part of that other issue. Best to work one step at a time;)